ibm/service/power/data_source_ibm_pi_catalog_images.go

```diff
@@ -120,13 +137,13 @@ func dataSourceIBMPICatalogImagesRead(ctx context.Context, d *schema.ResourceDat
 		return diag.FromErr(err)
 	}
 
-	cloudInstanceID := d.Get(helpers.PICloudInstanceId).(string)
+	cloudInstanceID := d.Get(Arg_CloudInstanceID).(string)
 	includeSAP := false
-	if s, ok := d.GetOk("sap"); ok {
+	if s, ok := d.GetOk(Arg_SAP); ok {
```

Comment:
While at it can you please set the Default attribute in the schema to `false`? change these lines....

```suggestion
	includeSAP := d.Get(Arg_SAP).(bool)
```
Same for `includeVTL`

Reply:
you mean like here right?

```
                        Arg_SAP: {
                                Default: false,
				Description: "Set true to include SAP images. The default value is false.",
				Optional:    true,
				Type:        schema.TypeBool,
			},
			Arg_VTL: {
                                Default: false,
				Description: "Set true to include VTL images. The default value is false.",
				Optional:    true,
				Type:        schema.TypeBool,
			},
```

I thought that if you didn't specify the default for a bool it would be false.

@yussufsh